.github/workflows/ci.yaml

```diff
@@ -18,12 +18,12 @@ jobs:
     name: unit tests
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - name: Set up Python 
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.9
+          python-version: '3.10'
```

Comment:
small point here: we're only testing on 3.10, but as a command line CLI that doesn't ship with its own python in a docker image (i.e. unlike the ETL) - we should probably test on all supported pythons

Doesn't need to happen on this PR - I filed #174 for it.